src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -992,11 +987,11 @@ function mapDrillIntersectionElement(header: ILegacyHeader, afm: AFM.IAfm): IDri
 
 export function getDrillIntersection(
     stackByItem: any,
-    viewByItem: any,
+    viewByItems: any,
```

Comment:
`any[]`

Reply:
Please also fix all calls of `getDrillIntersection()` function - there are 2 test cases passing one object instead of array of objects. I wonder how it is possible that those tests did not fail  😕

Reply:
Fixed :)